crates/net/network/src/transactions/config.rs

```diff
@@ -23,6 +25,9 @@ pub struct TransactionsManagerConfig {
     /// How new pending transactions are propagated.
     #[cfg_attr(feature = "serde", serde(default))]
     pub propagation_mode: TransactionPropagationMode,
+    /// Determines the policy for filtering incoming transaction annoucements based on `TxType`.
+    #[cfg_attr(feature = "serde", serde(default))]
+    pub announcement_filter_kind: AnnouncementFilterKind,
```

Comment:
do we need this here?
I see this as more like strict programmatically only setting